packages/api/tests/lambdas/test-publish-granules.js

```diff
@@ -103,3 +108,37 @@ test.serial('The publish-granules Lambda function takes a DynamoDB stream event
 
   t.is(Messages.length, 2);
```

Comment:
Shouldn't we test that the `event` is being set correctly? Something like 
```
t.is(Messages[0].event, 'Create');
t.is(Messages[1].event, 'Update');
```

Reply:
I see previous tests do test for `Update`, but nothing for `Create`.

Reply:
Sure I'll add it.